FirstDemoAPI/BookValidator.cs

```diff
@@ -0,0 +1,20 @@
+﻿using FirstDemoAPI.Model;
+using System;
+using System.Linq;
+
+namespace FirstDemoAPI
+{
+    public class BookValidator
+    {
+        public static string Check(Book book)
+        {
```

Comment:
What happens when both Name Author and Price is invalid?

Reply:
It will only respond as "Invalid Price, Price cannot be negative." .
As I am validating price first.
Should I have to change it and make it respond as "Both Price and Author name is Invalid" ?

Reply:
done the changes specified